pkg/forwarder/sync_forwarder.go

```diff
@@ -41,7 +45,18 @@ func (f *SyncForwarder) Stop() {
 func (f *SyncForwarder) sendHTTPTransactions(transactions []*transaction.HTTPTransaction) error {
 	for _, t := range transactions {
 		if err := t.Process(context.Background(), f.client); err != nil {
-			log.Errorf("SyncForwarder.sendHTTPTransactions: %s", err)
+			log.Errorf("SyncForwarder.sendHTTPTransactions first attempt: %s", err)
+
+			// If there is an error, instantiate a new HTTP client and retry one time
+			// The new HTTP client is instantiated in case the connection has been closed
+			log.Debug("Retrying transaction")
+			f.client = &http.Client{
+				Timeout:   forwarderTimeout,
+				Transport: utilhttp.CreateHTTPTransport(),
+			}
+			if err:= t.Process(context.Background(), f.client); err != nil {
+				log.("SyncForwarder.sendHTTPTransactions second attempt: %s", err)
+			}
```

Comment:
Since we're writing an error L48 mentioning the first attempt error, in case of success here, I would write an Info log mentioning that the retry has succeed.